### DISCUSSION TITLE
No more data

### DISCUSSION THREAD

**Discussion by Julien4135** [tillsteinbach/VWsFriend#427] [Q&A]
Hi, maybe there has been a change in the VW programs, but I have no data coming back from the car since 14/02/2023 ?

Do you think everything is working fine ? 

Thank you for your help ;)

**Comment by tillsteinbach** [comment]
Did you accept the new terms and conditions?

**Reply by Julien4135** [reply]
A no I don't think so?

**Reply by Julien4135** [reply]
I just did it on my country's official website, I got the thing via the app for my Tiguan Hybrid but not for the ID.3. Hopefully all will go up now. Thanks a lot for your help!
